pkg/operation/botanist/component/logging/kube_rbac_proxy.go

```diff
@@ -31,15 +31,16 @@ import (
 )
 
 const (
-	// PromtailName is the name used for labeling the Kubernetes resources associated with Promtail installed on the shoot nodes.
-	PromtailName = "gardener-promtail"
-	// PromtailRBACName is the name of the user used by promtail to auth gains Kube-RBAC-Proxy
-	PromtailRBACName = "gardener.cloud:logging:promtail"
-
 	managedResourceName = "shoot-node-logging"
 
 	kubeRBACProxyName            = "kube-rbac-proxy"
```

Comment:
Unrelated to your PR but why do we need this kubeRBACProxy again for the shoot-node-logging?

Reply:
I suggest to follow up with @vlvasilev about this question out of band.